### PR TITLE
Change plugin name

### LINKED ISSUE #22: Rename to match current naming convention
Consider renaming the plugin to `membrane_raw_audio_format`

### PR DESCRIPTION
**Pull request by dimamik**
Closes #22

### CODE REVIEW

**Comment by dimamik** [comment]
Probably GitHub repo name also needs to be changed

File: README.md [hunk 1]
```diff
@@ -21,8 +21,11 @@ reason, just add the following line to your `deps` in the `mix.exs` and run
 `mix deps.get`.
 
 ```elixir
-{:membrane_caps_audio_raw, "~> 0.6.0"}
+{:membrane_raw_audio_format, "~> 0.6.1"}
```
Comment (mickel8) [inline README.md:24]: We need to update badges too so 
```
[![Hex.pm](https://img.shields.io/hexpm/v/membrane_raw_audio_format.svg)](https://hex.pm/packages/membrane_raw_audio_format)
[![API Docs](https://img.shields.io/badge/api-docs-yellow.svg?style=flat)](https://hexdocs.pm/membrane_raw_audio_format)
[![CircleCI](https://circleci.com/gh/membraneframework/membrane_raw_audio_format/tree/master.svg?style=svg)](https://circleci.com/gh/membraneframework/membrane_raw_audio_format/tree/master)
```

and the link at the end

```
[![Software Mansion](https://logo.swmansion.com/logo?color=white&variant=desktop&width=200&tag=membrane-github)](https://swmansion.com/?utm_source=git&utm_medium=readme&utm_campaign=membrane_raw_audio_format)
```

File: README.md [hunk 2]
```diff
@@ -21,8 +21,11 @@ reason, just add the following line to your `deps` in the `mix.exs` and run
...
 ```
+
+
+
 ## Copyright and License
 
 Copyright 2018, [Software Mansion](https://swmansion.com/?utm_source=git&utm_medium=readme&utm_campaign=membrane)
```
Comment (mickel8) [inline README.md:31]:
```suggestion
Copyright 2018, [Software Mansion](https://swmansion.com/?utm_source=git&utm_medium=readme&utm_campaign=membrane_raw_audio_format)
```

File: mix.exs [hunk 3]
```diff
@@ -21,7 +21,7 @@ defmodule Membrane.Caps.Audio.Raw.Mixfile do
   defp docs do
     [
       main: "readme",
-      extras: ["README.md", "LICENSE"],
+      extras: ["README.md", LICENSE: [title: "License"]],
```
Comment (mickel8) [inline mix.exs:24]: We haven't used `title` option so far so I would leave it as it was :thinking:

File: mix.exs [hunk 4]
```diff
@@ -1,12 +1,12 @@
 defmodule Membrane.Caps.Audio.Raw.Mixfile do
```
Comment (mickel8) [inline mix.exs:1]:
```suggestion
defmodule Membrane.RawAudio.Mixfile do
```

File: mix.exs [hunk 5]
```diff
@@ -21,7 +21,7 @@ defmodule Membrane.Caps.Audio.Raw.Mixfile do
   defp docs do
     [
       main: "readme",
-      extras: ["README.md", "LICENSE"],
+      extras: ["README.md", LICENSE: [title: "License"]],
       formatters: ["html"],
       source_ref: "v#{@version}",
       nest_modules_by_prefix: [Membrane.Caps]
```
Comment (mickel8) [inline mix.exs:27]:
```suggestion
      nest_modules_by_prefix: [Membrane.RawAudio]
```

File: test/membrane_raw_audio_test.exs [hunk 6]
```diff
@@ -1,6 +1,6 @@
 defmodule Membrane.RawAudioTest do
   use ExUnit.Case, async: true
-  alias Membrane.Caps.Audio.Raw, as: RawAudio
+  alias Membrane.RawAudio, as: RawAudio
```
Comment (mickel8) [inline test/membrane_raw_audio_test.exs:3]:
```suggestion
  alias Membrane.RawAudio
```

File: test/membrane_raw_audio/format_spec.exs [hunk 7]
```diff
@@ -1,7 +1,7 @@
-defmodule Membrane.Caps.Audio.Raw.FormatTest do
+defmodule Membrane.RawAudio.FormatTest do
   use ExUnit.Case, async: true
 
-  alias Membrane.Caps.Audio.Raw.Format, as: RawAudio
+  alias Membrane.RawAudio.Format, as: Format
```
Comment (mickel8) [inline test/membrane_raw_audio/format_spec.exs:4]:
```suggestion
  alias Membrane.RawAudio.Format
```

**Review by mickel8 — CHANGES_REQUESTED** [review]

File: mix.exs (hunk 3 above)
Comment (mickel8) [reply mix.exs:24]: Maybe except,`membrane_raw_video_format` :thinking:

**Review by mickel8 — APPROVED** [review]

**Pull request merged**